test-noaho.py

```diff
@@ -402,3 +402,12 @@ def test_bad_mapped_trie():
 
         with pytest.raises(AssertionError):
             Mapped(path)
+
+def test_mapped_trie_payload():
+    trie = NoAho()
+    trie.add("foo", None)
+    trie.compile()
+    with tempfile.TemporaryDirectory(prefix="noahong-") as tmpdir:
+        path = os.path.join(tmpdir, "mapped")
+        with pytest.raises(PayloadWriteError):
+            trie.write(path)
```

Comment:
GitHub is surprised to see that your files does not end with a newline! Is that the default behavior of VS Code ?